src/views/ApartmentApp/components/StudentApplication/components/ApplicantList/components/ApplicantListItem/index.js

```diff
@@ -0,0 +1,154 @@
+import React, { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import {
+  Grid,
+  Divider,
+  ListItem,
+  ListItemAvatar,
+  ListItemSecondaryAction,
+  ListItemText,
+  Avatar,
+  IconButton,
+} from '@material-ui/core';
+import ClearIcon from '@material-ui/icons/Clear';
+import PersonIcon from '@material-ui/icons/Person';
+import StarBorderIcon from '@material-ui/icons/StarBorder';
+import StarIcon from '@material-ui/icons/Star';
+import user from '../../../../../../../../services/user';
+
+/**
+ * @typedef { import('../../../../services/user').StudentProfileInfo } StudentProfileInfo
+ */
+
+// Based off src/views/PeopleSearch/components/PeopleSearchResult
+// but using props.profile of type StudentProfileInfo
+// rather than using Person of type PeopleSearchResult
+const ApplicantListItem = ({
+  disabled,
+  profile,
+  isApplicationEditor,
+  onChangeEditor,
+  onApplicantRemove,
+}) => {
+  const [avatar, setAvatar] = useState(null);
+  const [hasNickName, setHasNickname] = useState(false);
+  const [personClass, setPersonClass] = useState(profile.Class);
+
+  useEffect(() => {
+    loadAvatar(profile);
+    createNickname(profile);
+    if (String(profile.PersonType).includes('stu') && profile.Class !== undefined) {
+      setPersonClass(profile.Class);
+    } else {
+      // Technically, this case should never happen because the list does not allow the user to add a non-student to the applicant list
+      setPersonClass('');
+    }
+  }, [profile]);
+
+  /**
+   * Creates the Avatar image of the given user
+   * @param {StudentProfileInfo} profile The StudentProfileInfo object for the student represented by this list item
+   */
+  const loadAvatar = async (profile) => {
+    try {
+      const { def: defaultImage, pref: preferredImage } = await user.getImage(profile.AD_Username);
+      setAvatar(preferredImage || defaultImage);
+    } catch {
+      setAvatar(
+        <svg width="50" height="50" viewBox="0 0 50 50">
+          <rect width="50" height="50" rx="10" ry="10" fill="#CCC" />
+        </svg>,
+      );
+    }
+  };
+
+  /**
+   * Saves the nickname of the given user if available
+   * @param {StudentProfileInfo} profile The StudentProfileInfo object for the student represented by this list item
+   */
+  const createNickname = (profile) => {
+    let Name = String(profile.fullName);
```

Comment:
The `fullName` property is added to a user's profile by the `formatName` function in `user.js`, called when getting a user's Profile. That means that when you use a Profile straight from the backend (i.e. when it is loaded as part of an Applicant), this property does not exist, so you end up with `undefined (nickname)` as their name - e.g.:
![image](https://user-images.githubusercontent.com/35319956/113884373-e8b00f00-978c-11eb-9f47-52d59d1dc03d.png)
The easiest fix is to probably find all references to `fullName` in your code and switch them over to use 
```
`${Profile.FirstName} ${Profile.LastName}`
```